apps/protocol-api/src/main.ts

```diff
@@ -71,13 +71,15 @@ const permissions = shield(
       guilds: or(isAuthenticated, hasToken),
       listUserByAddress: isAuthenticated,
       getContributionCountByDateForUserInRange: or(isAuthenticated, hasToken),
+      getContributionCountForDaoByActivityTypeInRange: or(isAuthenticated, hasToken),
```

Comment:
Would it be better to make this a more generic name. Like getActivityTypeContributionCount. And have the dao part of the name and in range part be arguments set by the client so we want reuse the logic. Trying to avoid having a bunch of different methods for slightly different functionality

Reply:
good idea

Reply:
Just renaming for the purposes of this PR. added Pro-551 for making the logic totally generic (tolerating nullable fields)